frontend/src/app/OrganizationView/OrganizationHomeContainer.jsx

```diff
@@ -24,13 +26,27 @@ const OrganizationHomeContainer = () => {
         <Route
           path={`${match.path}/queue`}
           render={() => {
-            return <TestQueue />;
+            return (
+              <AppInsightsErrorBoundary
```

Comment:
Can this just be done above the `Switch` level so it doesn't need to be repeated? Or do we lose some detail?